### PR TITLE
Slash command for ,include and ,exclude

### PR DESCRIPTION
**Pull request by taahamahdi**

### CODE REVIEW

File: src/commands/game_options/add.ts [hunk 1]
```diff
@@ -281,7 +291,7 @@ export default class AddCommand implements BaseCommand {
                                     }
                                 ),
                         },
-                        interaction
+                        matchedGroups.length === 0 ? interaction : undefined
```
Comment (Brainicism) [inline src/commands/game_options/add.ts:294]: Why no interaction response for `matchedGroups.length === 0`?

File: src/commands/game_options/add.ts [hunk 2]
```diff
@@ -352,17 +395,30 @@ export default class AddCommand implements BaseCommand {
...
+                                        allowOrPrevent:
+                                            LocalizationManager.localizer.translate(
+                                                messageContext.guildID,
+                                                "misc.failure.groupsExcludeConflict.prevent"
+                                            ),
                                     }
                                 ),
                         },
-                        interaction
+                        matchedGroups.length === 0 ? interaction : undefined
```
Comment (Brainicism) [inline src/commands/game_options/add.ts:421]: Same here

File: src/commands/game_options/exclude.ts [hunk 3]
```diff
@@ -83,78 +94,225 @@ export default class ExcludeCommand implements BaseCommand {
...
-                            message.guildID,
-                            "misc.failure.unrecognizedGroups.solution",
-                            {
-                                command: `\`${process.env.BOT_PREFIX}add exclude\``,
-                            }
-                        ),
-                    }
-                ),
+                description: `${descriptionText}\n\n${suggestionsText || ""}`,
                 footerText: excludeWarning,
```
Comment (Brainicism) [inline src/commands/game_options/exclude.ts:316]: Missing interaction here?

File: src/commands/game_options/groups.ts [hunk 4]
```diff
@@ -206,29 +229,33 @@ export default class GroupsCommand implements BaseCommand {
...
             );
 
-            await sendErrorMessage(MessageContext.fromMessage(message), {
+            await sendErrorMessage(messageContext, {
                 title: LocalizationManager.localizer.translate(
-                    message.guildID,
+                    messageContext.guildID,
                     "misc.failure.unrecognizedGroups.title"
                 ),
                 description: `${descriptionText}\n\n${suggestionsText || ""}`,
```
Comment (Brainicism) [inline src/commands/game_options/groups.ts:261]: Missing interaction here?

File: src/commands/game_options/include.ts [hunk 5]
```diff
@@ -82,124 +94,288 @@ export default class IncludeCommand implements BaseCommand {
...
-                            message.guildID,
-                            "misc.failure.unrecognizedGroups.solution",
-                            {
-                                command: `\`${process.env.BOT_PREFIX}add include\``,
-                            }
-                        ),
-                    }
-                ),
+                description: `${descriptionText}\n\n${suggestionsText || ""}`,
                 footerText: includeWarning,
```
Comment (Brainicism) [inline src/commands/game_options/include.ts:272]: Missing interaction here?

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: src/commands/game_options/add.ts (hunk 1 above)
Comment (taahamahdi) [reply src/commands/game_options/add.ts:294]: The interaction is only passed when no groups are matched so this error message acknowledges the interaction. When at least one group is matched, then the options message acknowledges it. In short, this prevents duplicate ACKs

**Review by Brainicism — APPROVED** [review]

**Pull request merged**